modules/telco-core-scheduling.adoc

```diff
@@ -10,17 +10,28 @@ New in this release::
+However, there are few specific use cases described in the following section.
+
 NUMA-aware scheduling can be enabled through the NUMA Resources Operator.
 For more information, see "Scheduling NUMA-aware workloads".
 
 Limits and requirements::
-* The default scheduler does not understand the NUMA locality of workloads. It only knows about the sum of all free resources on a worker node. This might cause workloads to be rejected when scheduled to a node with the topology manager policy set to `single-numa-node` or `restricted`.
-** For example, consider a pod requesting 6 CPUs and being scheduled to an empty node that has 4 CPUs per NUMA node. The total allocatable capacity of the node is 8 CPUs and the scheduler will place the pod there. The node local admission will fail, however, as there are only 4 CPUs available in each of the NUMA nodes.
-** All clusters with multi-NUMA nodes are required to use the NUMA Resources Operator. Use the `machineConfigPoolSelector` field in the `KubeletConfig` CR to select all nodes where NUMA aligned scheduling is needed.
-* All machine config pools must have consistent hardware configuration for example all nodes are expected to have the same NUMA zone count.
+* The default scheduler does not understand the NUMA locality of workloads.
+It only knows about the sum of all free resources on a worker node.
+This might cause workloads to be rejected when scheduled to a node with the topology manager policy set to single-numa-node or restricted.
+For more information, see "Topology Manager policies".
+** For example, consider a pod requesting 6 CPUs and being scheduled to an empty node that has 4 CPUs per NUMA node.
+The total allocatable capacity of the node is 8 CPUs. The scheduler places the pod on the empty node.
+The node local admission fails, as there are only 4 CPUs available in each of the NUMA nodes.
+* All clusters with multi-NUMA nodes are required to use the NUMA Resources Operator.
+See "Installing the NUMA Resources Operator" for more information.
+Use the `machineConfigPoolSelector` field in the `KubeletConfig` CR to select all nodes where NUMA aligned scheduling is required.
+* All machine config pools mux`st have consistent hardware configuration.
+For example, all nodes are expected to have the same NUMA zone count.
 
 Engineering considerations::
-* Pods might require annotations for correct scheduling and isolation. For more information on annotations, see "CPU partitioning and performance tuning".
-
-* You can configure SR-IOV virtual function NUMA affinity to be ignored during scheduling by using the `excludeTopology` field in `SriovNetworkNodePolicy` CR.
+* Pods might require annotations for correct scheduling and isolation.
+For more information about annotations, see "CPU partitioning and performance tuning".
+* You can configure SR-IOV virtual function NUMA affinity to be ignored during scheduling by using the excludeTopology field in `SriovNetworkNodePolicy` CR.
```

Comment:
I'm for leaving the markup as the word is a  [system item](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference).

```suggestion
* You can configure SR-IOV virtual function NUMA affinity to be ignored during scheduling by using the `excludeTopology` field in `SriovNetworkNodePolicy` CR.
```